interface/new/new_comprehensive_save.php

```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
```

Comment:
Do we know why this was failing? I just tried that SQL and it returned the expected results